packages/runner-shared/src/iframe/iframe-model.js

```diff
@@ -216,7 +218,28 @@ export class IframeModel {
     this.state.messageType = 'warning'
   }
 
-  _storeOriginalState () {
+  _storeOriginalState (snapshotUrl) {
+    if (!this.isAUTSameOrigin()) {
+      const Cypress = eventManager.getCypress()
+
+      // go get the final cross origin snapshot. Do this optimistically while we render the selected snapshot
+      Cypress.primaryOriginCommunicator.toAllSpecBridges('generate:final:snapshot', snapshotUrl)
+      Cypress.primaryOriginCommunicator.once('final:snapshot:generated', (finalSnapshot) => {
+        this.originalState = {
+          body: finalSnapshot.body,
+          htmlAttrs: finalSnapshot.htmlAttrs,
+          snapshot: finalSnapshot,
+          url: this.state.url,
+          // TODO: use same attr for both runner and runner-ct states.
+          // these refer to the same thing - the viewport dimensions.
```

Comment:
What attr is currently used?

Reply:
Honestly, I am not too sure. I retained the comment from the previous code here since now we set the state in two places. I figured it pertained to some of the state attributes for e2e and component testing but after a quick glance it looks like a lot of the state properties are calculated the same. Since this code should be unreachable for component testing, I am wondering if we even need this comment here.

Reply:
I think this comment can be removed, the runner-ct package is destined for removal anyway, since now E2E and Component tests use the same. @lmiller1990 can probably confirm, it looks to me like the comment actually would have been fine to remove in [this commit](https://github.com/cypress-io/cypress/commit/91a11dcff43f39f7625fbc1c695436912a37e831#diff-a3d08acaf125f704207748970036333b32b357cff5671c7179372642aa2bb21cR263-R264)? Since we weren't referencing `width` and `height` from `state` any more. 

Or Lachlan can tell us if this does relate to something that still needs doing?

Reply:
Oops, looks like I jumped the gun a bit - the `runner-ct` package is very much still in use on develop, where this code will end up first. I just happened across this comment in skimming my emails. I still think the TODO can be removed, since do now have a single runner on 10.0-release that will be merged soon.

Reply:
@marktnoonan I appreciate your input here. If this is the case, I should probably remove the TODO in both places in `runner-ct` then?